### PR TITLE
Transaction component

### LINKED ISSUE #38: TransactionList component
## `<TransactionList/>`

Iterates through the transactions and hands rendering responsibility to child `<Transaction/>` components. Should have an empty state for no transactions.

Other possibilities include pagination, infinite scrolling with lazy loading, or filtering by sender.

This component should be created in Storybook, with designs TBD.

### Input props

- `transactions[]`
	- `from`: the address from which the transaction was sent
	- `to`: address from which the transaction was sent
	- `amount`: the amount of currency moved in the transaction
	- `time`: the timestamp at which the transaction occurred
	- `trust`: the number of confirmations that the transaction has received

### PR DESCRIPTION
**Pull request by peasandwell**
### Description

Updated the Transaction and TransactionList components to have some basic styles, alongside with their Storybook counterparts.

### Issues fixed

#38 and #39

### Checklist

- [x] `npm test` returns a single error on the TransactionList container, but it was like that when I got here.

### CODE REVIEW

File: src/components/TransactionsList/TransactionsList.js [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
+				<Transaction
+					key={transaction.transactionHash}
+					{...transaction}
+				/>
+			))
+		}
+		{transactions.length <= 0 &&
+			<p className="no-results">
+		]		No transactions found!
+				<span className="icon">😖</span>
```
Comment (ShaunBulbrook) [inline src/components/TransactionsList/TransactionsList.js]: Emoji in the code!?!?!?

File: src/components/TransactionsList/TransactionsList.js [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
+			transactions.map(transaction => (
+				<Transaction
+					key={transaction.transactionHash}
+					{...transaction}
+				/>
+			))
+		}
+		{transactions.length <= 0 &&
+			<p className="no-results">
+		]		No transactions found!
```
Comment (ShaunBulbrook) [inline src/components/TransactionsList/TransactionsList.js]: Please remove errant bracket.

File: src/components/TransactionsList/TransactionsList.js (hunk 1 above)
Comment (peasandwell) [reply src/components/TransactionsList/TransactionsList.js]: Please see this site for emoji support on relevant browsers - http://caniemoji.com/

File: src/components/TransactionsList/TransactionsList.js (hunk 2 above)
Comment (peasandwell) [reply src/components/TransactionsList/TransactionsList.js]: I have now removed errant bracket.

**Comment by dan1elhughes** [comment]
Please rebase onto master, then 👍

**Pull request merged**